Cargo.toml

```diff
@@ -5,7 +5,7 @@ repository = "https://github.com/kata-containers/cgroups-rs"
 keywords = ["linux", "cgroup", "containers", "isolation"]
 categories = ["os", "api-bindings", "os::unix-apis"]
 license = "MIT OR Apache-2.0"
-version = "0.2.0"
+version = "0.2.1"
```

Comment:
shall we increase version per PR ? @lifupan @bergwolf

Reply:
I need use this feature immediately, so the version should be increase.

Reply:
For the version mechanism I think we do not need to bump the version for every pr, We should accumulate some features to release a new version, but for every critical bug fix we need release a new version ASAP